lib/appraisal/bundler_dsl.rb

```diff
@@ -25,7 +25,11 @@ def run(&block)
     end
 
     def gem(name, *requirements)
-      @dependencies.add(name, substitute_git_source(requirements))
+      if requirements.include?(:remove)
```

Comment:
If we go with `remove_gem`, we wouldn't have a conditional in this method anymore. Instead we would introduce a new method:

```rb
def remove_gem(name)
  @dependencies.remove(name)
end
```

This also has the advantage of leaving off the `*requirements` argument when removing a gem, since we would have ignored most of those anyway.

Reply:
This is much cleaner. I've gone ahead and implemented this just how you suggested. It's much simpler and easier to use.